VirtoCommerce.Platform.Web/Modularity/ManifestModuleCatalog.cs

```diff
@@ -170,25 +170,50 @@ public override void Validate()
             return result;
         }
 
+        private static void CopyLocalizationAssemblies(string sourceDirectoryPath, string targetDirectoryPath)
```

Comment:
Can we name this just "CopyAssamblies"?

Reply:
I don't think so. This method copy only resource localisation assemblies. Method CopyAssemblyFiles copy all other assemblies.